src/Components/App/App.tsx

```diff
@@ -1,23 +1,22 @@
+import React, { Component } from 'react'
+
+import Navbar from '../Navbar/Navbar'
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
+  constructor(props: {}) {
+    super(props)
+    this.state = {
+    }
+  }
+
+  render() {
+    return (
+      <main className="App">
+        <Navbar />
+      </main>
+    )
+  }
 }
 
 export default App;
```

Comment:
Looks like we have the basic architecture setup. Shouldn't be too difficult adding TypeScript and Router.